issues.go

```diff
@@ -322,7 +322,7 @@ type ListProjectIssuesOptions struct {
 	NotLabels          *Labels          `url:"not[labels],comma,omitempty" json:"not[labels],omitempty"`
 	WithLabelDetails   *bool            `url:"with_labels_details,omitempty" json:"with_labels_details,omitempty"`
 	Milestone          *string          `url:"milestone,omitempty" json:"milestone,omitempty"`
-	NotMilestone       *[]string        `url:"not[milestone],omitempty" json:"not[milestone],omitempty"`
+	NotMilestone       *string          `url:"not[milestone],omitempty" json:"not[milestone],omitempty"`
```

Comment:
I think you should correct the other ones as this one was the correct one...

Reply:
I finally think you are right: all (Labels, Milestones ...) seem to be slices. But I've never use it like that. I'll do some more tests.  
Feel free to reject that PR if you wish. After my tests, I'll be back with another one.

Reply:
I've done some more tests. `NotLabels` and others ones except `NotMilestone` accept slice of strings.

    /api/v4/projects/100/issues?not[labels]=label1,label3

Since an issue can only be related to one milestone, `Milestone` in ListXXXIssuesOptions structure is definitely a (simple) string.  But `NotMilestone` could have been a slice of string. Unfortunately it does not work. The API seems to accept a string only.

    /api/v4/projects/100/issues?not[milestone]=milestone2

In conclusion you are allowed to give exactly one milestone in `NotMilestone` field.

I think my Pull request is stile valid.

Reply:
Check... Thanks for going the extra mile on this one!